src/provider/shplonk.rs

```diff
@@ -93,12 +90,11 @@ where
     C: &Commitment<NE>,
     polys: &[Vec<E::Fr>],
   ) -> Vec<E::G1Affine> {
-    // TODO avoid computing commitment to constant polynomial
     let mut comms: Vec<NE::GE> = (1..polys.len())
       .into_par_iter()
       .map(|i| <NE::CE as CommitmentEngineTrait<NE>>::commit(ck, &polys[i]).comm)
       .collect();
-    // TODO avoid inserting commitment known to verifier
+
     comms.insert(0, C.comm);
```

Comment:
This should be removed so that it does not appear in `pi` or get added to the transcript. It must still however be part of the RLC with `q` when computing the batched commitment.